zerver/tests/test_openapi.py

```diff
@@ -20,6 +23,14 @@
 TEST_RESPONSE_BAD_REQ = '400'
 TEST_RESPONSE_SUCCESS = '200'
 
+VARMAP = {
+    'integer': int,
+    'string': str,
+    'boolean': bool,
+    'array': list,
+    'Typing.List': list,
+    'NoneType': None,
+}
```

Comment:
Just to be explicit, have you verified that if you edit something to have an incorrect type, this test fails?

Reply:
Yup, I tested it out.

Reply:
Also added a sample error message in a comment below this.